contents/security/sqlInjection.md

```diff
@@ -13,86 +13,132 @@
 
 # SQL Injection
 
-Author: [Lewis Koh](https://github.com/nus-cs3281/2018/blob/master/students/lewisKoh/lewisKoh-Resume.md)
+**Authors: [Jiang Chunhui](https://github.com/Adoby7), [Lewis Koh](https://github.com/nus-cs3281/2018/blob/master/students/lewisKoh/lewisKoh-Resume.md)**
 
-## Overview
+**Reviewers: [Ayush Chatto](https://github.com/AyushChatto), [Monika Manuela Hengki](https://github.com/monmanuela),  [Nicholas Chua](https://github.com/nicholaschuayunzhi), [Rachael Sim](https://github.com/rachx), [Tran Tien Dat](https://github.com/tran-tien-dat), [Wen Xin](https://github.com/wenmogu)**
 
-SQL (Structured Query Language) Injection is a vulnerability found in websites using databases.
-
-SQL is a common language which is used by websites to communicate with databases. Databases can be used to store persistent data, such as usernames and passwords, sensitive account data, or other important information used by the website. Databases are usually made of many "tables" 
-which are organised in rows and columns. Each row is a separate entry in the table, and each column is a specific parameter which can be used by the entry.
-
-As an example, a table in a database may look like this:
+## SQL
+SQL (Structured Query Language) is a common language which is used by websites to communicate with databases. Databases can be used to store persistent data, such as usernames and passwords, sensitive account data, or other important information used by the website. Typically, SQL works on relational databases, which are usually made of many "tables" organised in rows and columns. Each row is a separate entry in the table, and each column is a specific parameter which can be used by the entry. A sample table is shown below:
 
 |   UserId   |    Username   |Password|
 |:----------:|:-------------:|:------:|
 |      1     |     Admin     | 123456 |
 |      2     |     Alice     | pw1234 |
 |      3     |      ...      |   ...  |
 
-
 SQL is used to interact with the database by sending "queries" which the database responds to. Some common SQL commands used include: 
 * `SELECT` - retrieves information from a table
 * `UPDATE` - changes information from a table
 * `INSERT INTO` - adds a new entry into a table
 * `DELETE FROM` - deletes information from a table
+* `DROP` - deletes the whole table
+
+In addition, a query could use parameters to filter, reorder, and group the returned results. For example, the following query will only returns the records in table "users" whose user name is "Admin". Here the parameter `Username = Admin` in the `WHERE clause` works as a filter.
+```SQL
+SELECT * FROM users WHERE Username = 'Admin'
+```
+
+
+More information about SQL can be found [here](https://www.w3schools.com/sql/).
+
+## How does SQL Injection Work?
+
+**SQL injection is the placement of malicious code in SQL statements, via web page input.**
```

Comment:
This is a verbatim quote from w3schools page. You should cite the source right here.